test/LightQuery.Shared.Tests/QueryableProcessorTests.cs

```diff
@@ -13,6 +13,83 @@ private class User
             public string Email { get; set; }
         }
 
+        #region Relational Data Sorting
+        private class Order
+        {
+            public string Title { get; set; }
+            public Product Product { get; set; }
+        }
+
+        private class Product
+        {
+            public string Name { get; set; }
+            public int Quantity { get; set; }
+            public decimal Amount { get; set; }
+            public ProductDetail Detail { get; set; }
+        }
+
+        private class ProductDetail
+        {
+            public string Barcode { get; set; }
+            public string ManufacturerCode { get; set; }
+        }
+
+        // Relational Data Sorting Sample Data
+        private IQueryable<Order> GetOrders()
+        {
+            return new[]
```

Comment:
Could you append something like `.OrderBy(c => Guid.newGuid())`, just to ensure that it's not coming back in the order it's defined? The test below that checks if it's order by title would succeed even if no ordering is applied at all currently.

Reply:
```.OrderBy(x=> Guid.NewGuid())``` added 👍